bedrock/firefox/templates/firefox/built-for-you/includes/v1-pinocchio-de.html

```diff
@@ -0,0 +1,25 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<div class="c-hero-video">
+  <video preload="auto" playsinline autoplay muted loop poster="{{ static('img/firefox/built-for-you/pinocchio-1000.jpg') }}" width="800" height="800">
+    <source src="https://assets.mozilla.net/video/built-for-you/video-pinocchio-de.webm" type="video/webm">
+  </video>
+
+  {{ resp_img('img/firefox/built-for-you/pinocchio-800.jpg',
```

Comment:
**Nit:** (non-blocking) adding some some `sizes` info to these image might help browsers to ppick out the most appropriate size a bit better.